jsoo/events_js.ml

```diff
@@ -45,7 +45,7 @@ let update ~status t e =
   let chars =
     keys
     |> List.filter_map (fun key ->
-           match key with `input_char c -> Some c | _ -> None)
+        match key with `input_char c -> Some c | _ -> None)
```

Comment:
I told @art-w that I reduced indentation from 5 to 4, but it used to be SEVEN :rofl: